### PR TITLE
container config section: various markdown fixes

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
@usha-mandya

File: config/containers/logging/configure.md [hunk 1]
```diff
@@ -159,25 +172,26 @@ see more options.
...
+  third party logging solutions, including: 
+    - `syslog`
+    - `gelf`
+    - `fluentd`
+    - `awslogs`
+    - `splunk`
+    - `etwlogs`
+    - `gcplogs`
+    - `Logentries`
+- When using Docker Community Engine, the `docker logs` command is only available
```
Comment (usha-mandya) [inline config/containers/logging/configure.md:188]: Should we update this to Docker Engine?

Comment (thaJeztah) [reply config/containers/logging/configure.md:188]: This feature has been open-sourced by Mirantis (see https://github.com/moby/moby/pull/40543), so will be included in the upcoming Engine release; we'll have to rewrite some of these pages related to that, so I thought to do it as part of that

File: config/containers/logging/configure.md [hunk 2]
```diff
@@ -159,25 +172,26 @@ see more options.
...
     - `local`
     - `json-file`
     - `journald`
-
-- Reading log information requires decompressing rotated log files, which causes a temporary increase in disk usage (until the log entries from the rotated files are read) and an increased CPU usage while decompressing. 
-- The capacity of the host storage where docker’s data directory resides determines the maximum size of the log file information.
+- Reading log information requires decompressing rotated log files, which causes
+  a temporary increase in disk usage (until the log entries from the rotated
+  files are read) and an increased CPU usage while decompressing. 
+- The capacity of the host storage where docker’s data directory resides
```
Comment (usha-mandya) [inline config/containers/logging/configure.md:196]: How about `The capacity of the host storage where the Docker data directory resides determines the maximum size of the log file information.`

Comment (thaJeztah) [reply config/containers/logging/configure.md:196]: Ah, yes, I can do that

File: config/containers/logging/dual-logging.md [hunk 3]
```diff
@@ -6,47 +6,57 @@ title: Use docker logs to read container logs for remote logging drivers
...
+- syslog
+- gelf
+- fluentd
+- awslogs
+- splunk
+- etwlogs
+- gcplogs
+- logentries
+
+This created multiple problems, especially with UCP, when attempting to gather
```
Comment (usha-mandya) [inline config/containers/logging/dual-logging.md:23]: Can we remove the reference to UCP?

File: config/containers/logging/dual-logging.md [hunk 4]
```diff
@@ -6,47 +6,57 @@ title: Use docker logs to read container logs for remote logging drivers
...
+log data in an automated and standard way. Log information could only be accessed
+and viewed through the third-party solution in the format specified by that
+third-party tool. 
+
+Starting with Docker Engine Enterprise 18.03.1-ee-1, you can use `docker logs`
```
Comment (usha-mandya) [inline config/containers/logging/dual-logging.md:28]: Should we remove the text `Starting with Docker Engine Enterprise 18.03.1-ee-1,` altogether, or use a Community release which introduced this feature?

File: config/containers/logging/dual-logging.md [hunk 5]
```diff
@@ -6,47 +6,57 @@ title: Use docker logs to read container logs for remote logging drivers
...
+[Configure the default logging driver](/config/containers/logging/configure) for
+additional information. 
 
 
 ## Prerequisites 
  
-- Docker Enterprise - Dual logging is only supported for Docker Enterprise, and is enabled by default starting with 
-Engine Enterprise 18.03.1-ee-1.
+Dual logging is only supported for Docker Enterprise, and is enabled by default
+starting with Engine Enterprise 18.03.1-ee-1.
```
Comment (usha-mandya) [inline config/containers/logging/dual-logging.md:41]: Similar comment as above on removing references to EE releases?

File: config/containers/logging/dual-logging.md (hunk 4 above)
Comment (thaJeztah) [reply config/containers/logging/dual-logging.md:28]: See my other comment; it's not _yet_ in a Docker Community release, so I need to do a follow-up to prepare for the next release

File: config/containers/logging/splunk.md [hunk 6]
```diff
@@ -60,12 +62,12 @@ The following properties let you configure the splunk logging driver.
...
 | `splunk-token`              | required | Splunk HTTP Event Collector token.                                                                                                                                                                                                                                                                                                          |
-| `splunk-url`                | required | Path to your Splunk Enterprise, self-service Splunk Cloud instance, or Splunk Cloud managed cluster (including port and scheme used by HTTP Event Collector) in one of the following formats: `https://your_splunk_instance:8088` or `https://input-prd-p-XXXXXXX.cloud.splunk.com:8088` or `https://http-inputs-XXXXXXXX.splunkcloud.com`. |
+| `splunk-url`                | required | Path to your Splunk Enterprise, self-service Splunk Cloud instance, or Splunk Cloud managed cluster (including port and scheme used by HTTP Event Collector) in one of the following formats: `https://your_splunk_instance:8088`, `https://input-prd-p-XXXXXXX.cloud.splunk.com:8088`, or `https://http-inputs-XXXXXXXX.splunkcloud.com`.  |
```
Comment (usha-mandya) [inline config/containers/logging/splunk.md:65]: The table get cut in the browser view. Not sure if it's just my settings. Could you check?

image

File: config/containers/logging/configure.md (hunk 1 above)
Comment (usha-mandya) [reply config/containers/logging/configure.md:188]: Sure, that makes sense. Thanks.

File: config/containers/logging/splunk.md (hunk 6 above)
Comment (thaJeztah) [reply config/containers/logging/splunk.md:65]: It looks like the page isn't wide enough on 1280px wide

Screenshot 2020-04-08 at 13 55 50



but it is working on either a wider window, or with the right-hand navigation collapsed:

Screenshot 2020-04-08 at 13 56 01

I checked the same page on docs.docker.com, and it already has the same issue: https://docs.docker.com/config/containers/logging/splunk/

So it's not a regression at least

**Comment by thaJeztah** [comment]
@usha-mandya updated; removed the UCP reference, and applied your wording suggestion. The table-width issue is not new, so didn't change that

**Pull request merged**